src/brevitas_examples/common/learned_round/learned_round_optimizer.py

```diff
@@ -492,12 +496,10 @@ def _optimize_learned_round_block(
         if self.learn_scale and scale_params is not None:
             optimizer_scale = self.scale_optimizer_class(
                 scale_params,
-                lr=self.optimizer_scale_lr,
-                **self.optimizer_kwargs,
+                **self.scale_optimizer_kwargs,
             )
             lr_scheduler_scale = (
-                self.lr_scheduler_class(
-                    optimizer_scale, start_factor=1, end_factor=0, total_iters=600)
+                self.lr_scheduler_class(optimizer_scale, **self.lr_scheduler_kwargs)
```

Comment:
Should we allow for different LRSchedulers for the scale/learned round parameters?